src/aura/DragNDrop/DragNDrop.cmp

```diff
@@ -0,0 +1,30 @@
+<aura:component controller="DragNDropController"
```

Comment:
We usually append "Cmp" at the end of the name of lightning components so we can distinguish them in salesforce under lightning components.
So we have:

- ClickEvt.evt - lightning events
- MyApp.app - lightning app
- MyCmp.cmp - lightning component